packages/vite/src/node/plugins/asset.ts

```diff
@@ -418,11 +418,17 @@ export async function urlToBuiltUrl(
 // Inspired by https://github.com/iconify/iconify/blob/main/packages/utils/src/svg/url.ts
 function svgToDataURL(content: Buffer): string {
   const stringContent = content.toString()
+
+  const singleQuoteInDoubleQuotes = /"[^"']*'[^"]*"/
+  const doubleQuoteInSingleQuotes = /'[^'"]*"[^']*'/
```

Comment:
Would you merge these regexes into a single regex (`"[^"']*'[^"]*"|'[^'"]*"[^']*'`) so that we can make it a single call? Also would you move the regex to the top-level scope so that we don't need to construct the regex every time?